src/humancellatlas/data/metadata/api.py

```diff
@@ -762,15 +762,22 @@ class IpscInductionProtocol(Protocol):
 
 @dataclass(init=False)
 class ImagingProtocol(Protocol):
-    target: List[ImagingTarget]  # A list so all the ImagingTarget objects can be tallied when indexed
+    probe: List[ImagingProbe]  # A list so all the ImagingProbe objects can be tallied when indexed
 
     def __init__(self,
                  json: JSON,
                  metadata_manifest_entry: Optional[ManifestEntry]
                  ) -> None:
         super().__init__(json, metadata_manifest_entry)
         content = json.get('content', json)
-        self.target = [ImagingTarget.from_json(target) for target in content['target']]
+        self.probe = [ImagingProbe.from_json(probe)
+                      for probe in lookup(content, 'probe', 'target', default=[])]
+
+    @property
+    def target(self) -> str:
```

Comment:
Should the return type be `list[ImagingProbe]`?

Reply:
@danielsotirhos Ah I messed up it should be `List[Optional[ImagingProbe]]` because of the default=[] that can be provided from the comprehension.

Reply:
Good catch. In previous versions of the schema `target` (and then later `probe`) was a required field, however in the latest https://schema.humancellatlas.org/type/protocol/imaging/11.3.0/imaging_protocol `probe` became optional.

Reply:
> @danielsotirhos Ah I messed up it should be List[Optional[ImagingProbe]] because of the default=[] that can be provided from the comprehension.

This statement is incorrect, all containers that have uniform types are allowed to be empty. The first suggestion was correct.